<!DOCTYPE html>
<html lang="en" xmlns="http://www.w3.org/1999/xhtml">

<head>
    <meta charset="utf-8" />
    <title>Internal Server Error</title>
    <style>
        body {
            font-family: 'Segoe UI', Tahoma, Arial, Helvetica, sans-serif;
            font-size: .813em;
            color: #222;
            background-color: #fff;
        }

        h1,
        h2,
        h3,
        h4,
        h5 {
            /*font-family: 'Segoe UI',Tahoma,Arial,Helvetica,sans-serif;*/
            font-weight: 100;
        }

        h1 {
            color: #44525e;
            margin: 15px 0 15px 0;
        }

        h2 {
            margin: 10px 5px 0 0;
        }

        h3 {
            color: #363636;
            margin: 5px 5px 0 0;
        }

        code {
            font-family: Consolas, "Courier New", courier, monospace;
        }

        body .titleerror {
            padding: 3px 3px 6px 3px;
            display: block;
            font-size: 1.5em;
            font-weight: 100;
        }

        body .location {
            margin: 3px 0 10px 30px;
        }

        #header {
            font-size: 18px;
            padding: 15px 0;
            border-top: 1px #ddd solid;
            border-bottom: 1px #ddd solid;
            margin-bottom: 0;
        }

            #header li {
                display: inline;
                margin: 5px;
                padding: 5px;
                color: #a0a0a0;
                cursor: pointer;
            }

            #header .selected {
                background: #44c5f2;
                color: #fff;
            }

        #stackpage ul {
            list-style: none;
            padding-left: 0;
            margin: 0;
            /*border-bottom: 1px #ddd solid;*/
        }

        #stackpage .details {
            font-size: 1.2em;
            padding: 3px;
            color: #000;
        }

        #stackpage .stackerror {
            padding: 5px;
            border-bottom: 1px #ddd solid;
        }


        #stackpage .frame {
            padding: 0;
            margin: 0 0 0 30px;
        }

            #stackpage .frame h3 {
                padding: 2px;
                margin: 0;
            }

        #stackpage .source {
            padding: 0 0 0 30px;
        }

            #stackpage .source ol li {
                font-family: Consolas, "Courier New", courier, monospace;
                white-space: pre;
                background-color: #fbfbfb;
            }

        #stackpage .frame .source .highlight li span {
            color: #FF0000;
        }

        #stackpage .source ol.collapsible li {
            color: #888;
        }

            #stackpage .source ol.collapsible li span {
                color: #606060;
            }

        .page table {
            border-collapse: separate;
            border-spacing: 0;
            margin: 0 0 20px;
        }

        .page th {
            vertical-align: bottom;
            padding: 10px 5px 5px 5px;
            font-weight: 400;
            color: #a0a0a0;
            text-align: left;
        }

        .page td {
            padding: 3px 10px;
        }

        .page th,
        .page td {
            border-right: 1px #ddd solid;
            border-bottom: 1px #ddd solid;
            border-left: 1px transparent solid;
            border-top: 1px transparent solid;
            box-sizing: border-box;
        }

            .page th:last-child,
            .page td:last-child {
                border-right: 1px transparent solid;
            }

        .page .length {
            text-align: right;
        }

        a {
            color: #1ba1e2;
            text-decoration: none;
        }

            a:hover {
                color: #13709e;
                text-decoration: underline;
            }

        .showRawException {
            cursor: pointer;
            color: #44c5f2;
            background-color: transparent;
            font-size: 1.2em;
            text-align: left;
            text-decoration: none;
            display: inline-block;
            border: 0;
            padding: 0;
        }

        .rawExceptionStackTrace {
            font-size: 1.2em;
        }

        .rawExceptionBlock {
            border-top: 1px #ddd solid;
            border-bottom: 1px #ddd solid;
        }

        .showRawExceptionContainer {
            margin-top: 10px;
            margin-bottom: 10px;
        }

        .expandCollapseButton {
            cursor: pointer;
            float: left;
            height: 16px;
            width: 16px;
            font-size: 10px;
            position: absolute;
            left: 10px;
            background-color: #eee;
            padding: 0;
            border: 0;
            margin: 0;
        }
    </style>
</head>

<body>
    <h1>An unhandled exception occurred while processing the request.</h1>
    <div class="titleerror">
        TokenResponseException: Error:&quot;invalid_grant&quot;, Description:&quot;Invalid JWT:
        Token must be a short-lived token (60 minutes) and in a reasonable timeframe. Check your iat and exp values and
        use a clock with skew to account for clock differences between systems.&quot;, Uri:&quot;&quot;
    </div>
    <p class="location">
        Google.Apis.Auth.OAuth2.Requests.TokenRequestExtenstions&#x2B;&lt;ExecuteAsync&gt;d__0.MoveNext() in
        <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\Requests\TokenRequestExtenstions.cs">TokenRequestExtenstions.cs</code>,
        line 52
    </p>
    <ul id="header">
        <li id="stack" tabindex="1" class="selected">
            Stack
        </li>
        <li id="query" tabindex="2">
            Query
        </li>
        <li id="cookies" tabindex="3">
            Cookies
        </li>
        <li id="headers" tabindex="4">
            Headers
        </li>
    </ul>

    <div id="stackpage" class="page">
        <ul>
            <li>
                <h2 class="stackerror">
                    TokenResponseException: Error:&quot;invalid_grant&quot;,
                    Description:&quot;Invalid JWT: Token must be a short-lived token (60 minutes) and in a reasonable
                    timeframe. Check your iat and exp values and use a clock with skew to account for clock differences
                    between systems.&quot;, Uri:&quot;&quot;
                </h2>
                <ul>
                    <li class="frame" id="frame1">
                        <h3>
                            Google.Apis.Auth.OAuth2.Requests.TokenRequestExtenstions&#x2B;&lt;ExecuteAsync&gt;d__0.MoveNext()
                            in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\Requests\TokenRequestExtenstions.cs">TokenRequestExtenstions.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame2">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame3">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame4">
                        <h3>
                            Google.Apis.Auth.OAuth2.ServiceAccountCredential&#x2B;&lt;RequestAccessTokenAsync&gt;d__23.MoveNext()
                            in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\ServiceAccountCredential.cs">ServiceAccountCredential.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame5">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame6">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame7">
                        <h3>
                            Google.Apis.Auth.OAuth2.TokenRefreshManager&#x2B;&lt;RefreshTokenAsync&gt;d__12.MoveNext()
                            in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\TokenRefreshManager.cs">TokenRefreshManager.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame8">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame9">
                        <h3>
                            Google.Apis.Auth.OAuth2.TokenRefreshManager.ResultWithUnwrappedExceptions&lt;T&gt;(Task&lt;T&gt;
                            task) in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\TokenRefreshManager.cs">TokenRefreshManager.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame10">
                        <h3>System.Threading.Tasks.ContinuationResultTaskFromResultTask.InnerInvoke()</h3>

                    </li>
                    <li class="frame" id="frame11">
                        <h3>
                            System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback
                            callback, object state)
                        </h3>

                    </li>
                    <li class="frame" id="frame12">
                        <h3>System.Threading.Tasks.Task.ExecuteWithThreadLocal(ref Task currentTaskSlot)</h3>

                    </li>
                    <li class="frame" id="frame13">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame14">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame15">
                        <h3>
                            Google.Apis.Auth.OAuth2.TokenRefreshManager&#x2B;&lt;GetAccessTokenForRequestAsync&gt;d__10.MoveNext()
                            in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\TokenRefreshManager.cs">TokenRefreshManager.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame16">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame17">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame18">
                        <h3>
                            Google.Apis.Auth.OAuth2.ServiceAccountCredential&#x2B;&lt;GetAccessTokenForRequestAsync&gt;d__24.MoveNext()
                            in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\ServiceAccountCredential.cs">ServiceAccountCredential.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame19">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame20">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame21">
                        <h3>
                            Google.Apis.Auth.OAuth2.ServiceCredential&#x2B;&lt;InterceptAsync&gt;d__26.MoveNext() in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\ServiceCredential.cs">ServiceCredential.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame22">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame23">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame24">
                        <h3>
                            Google.Apis.Http.ConfigurableMessageHandler&#x2B;&lt;SendAsync&gt;d__63.MoveNext() in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Core\Http\ConfigurableMessageHandler.cs">ConfigurableMessageHandler.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame25">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame26">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame27">
                        <h3>
                            System.Runtime.CompilerServices.ConfiguredTaskAwaitable&#x2B;ConfiguredTaskAwaiter.GetResult()
                        </h3>

                    </li>
                    <li class="frame" id="frame28">
                        <h3>System.Net.Http.HttpClient&#x2B;&lt;FinishSendAsyncBuffered&gt;d__58.MoveNext()</h3>

                    </li>
                    <li class="frame" id="frame29">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame30">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame31">
                        <h3>
                            Google.Apis.Requests.ClientServiceRequest&#x2B;&lt;ExecuteUnparsedAsync&gt;d__30.MoveNext()
                            in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis\Requests\ClientServiceRequest.cs">ClientServiceRequest.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame32">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame33">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame34">
                        <h3>
                            Google.Apis.Requests.ClientServiceRequest&#x2B;&lt;ExecuteAsync&gt;d__27.MoveNext() in
                            <code title="C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis\Requests\ClientServiceRequest.cs">ClientServiceRequest.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame35">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame36">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame37">
                        <h3>Google.Apis.Requests.ClientServiceRequest&#x2B;&lt;ExecuteAsync&gt;d__26.MoveNext()</h3>

                    </li>
                    <li class="frame" id="frame38">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame39">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame40">
                        <h3>System.Runtime.CompilerServices.TaskAwaiter.GetResult()</h3>

                    </li>
                    <li class="frame" id="frame41">
                        <h3>
                            IntegrateWithCloud.Controllers.DefaultController&#x2B;&lt;ReadAsync&gt;d__7.MoveNext() in
                            <code title="D:\IntegrateWithCloud\IntegrateWithCloud\Controllers\DefaultController.cs">DefaultController.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame42">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame43">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame44">
                        <h3>
                            IntegrateWithCloud.Controllers.DefaultController&#x2B;&lt;Get2&gt;d__5.MoveNext() in
                            <code title="D:\IntegrateWithCloud\IntegrateWithCloud\Controllers\DefaultController.cs">DefaultController.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame45">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame46">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame47">
                        <h3>
                            IntegrateWithCloud.Controllers.DefaultController&#x2B;&lt;GetAsync&gt;d__4.MoveNext() in
                            <code title="D:\IntegrateWithCloud\IntegrateWithCloud\Controllers\DefaultController.cs">DefaultController.cs</code>
                        </h3>

                    </li>
                    <li class="frame" id="frame48">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame49">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame50">
                        <h3>lambda_method(Closure , object )</h3>

                    </li>
                    <li class="frame" id="frame51">
                        <h3>Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable&#x2B;Awaiter.GetResult()</h3>

                    </li>
                    <li class="frame" id="frame52">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker&#x2B;&lt;InvokeActionMethodAsync&gt;d__12.MoveNext()
                        </h3>

                    </li>
                    <li class="frame" id="frame53">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame54">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame55">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker&#x2B;&lt;InvokeNextActionFilterAsync&gt;d__10.MoveNext()
                        </h3>

                    </li>
                    <li class="frame" id="frame56">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame57">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext
                            context)
                        </h3>

                    </li>
                    <li class="frame" id="frame58">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(ref State next, ref Scope
                            scope, ref object state, ref bool isCompleted)
                        </h3>

                    </li>
                    <li class="frame" id="frame59">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker&#x2B;&lt;InvokeInnerFilterAsync&gt;d__14.MoveNext()
                        </h3>

                    </li>
                    <li class="frame" id="frame60">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame61">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame62">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker&#x2B;&lt;InvokeNextResourceFilter&gt;d__22.MoveNext()
                        </h3>

                    </li>
                    <li class="frame" id="frame63">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame64">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
                        </h3>

                    </li>
                    <li class="frame" id="frame65">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(ref State next, ref Scope scope, ref
                            object state, ref bool isCompleted)
                        </h3>

                    </li>
                    <li class="frame" id="frame66">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker&#x2B;&lt;InvokeFilterPipelineAsync&gt;d__17.MoveNext()
                        </h3>

                    </li>
                    <li class="frame" id="frame67">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame68">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame69">
                        <h3>
                            Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker&#x2B;&lt;InvokeAsync&gt;d__15.MoveNext()
                        </h3>

                    </li>
                    <li class="frame" id="frame70">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame71">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame72">
                        <h3>Microsoft.AspNetCore.Builder.RouterMiddleware&#x2B;&lt;Invoke&gt;d__4.MoveNext()</h3>

                    </li>
                    <li class="frame" id="frame73">
                        <h3>System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()</h3>

                    </li>
                    <li class="frame" id="frame74">
                        <h3>
                            System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task
                            task)
                        </h3>

                    </li>
                    <li class="frame" id="frame75">
                        <h3>
                            Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware&#x2B;&lt;Invoke&gt;d__7.MoveNext()
                        </h3>

                    </li>
                </ul>
            </li>
            <li>
                <br />
                <div class="rawExceptionBlock">
                    <div class="showRawExceptionContainer">
                        <button class="showRawException" data-exceptionDetailId="exceptionDetail1">Show raw exception details</button>
                    </div>
                    <div id="exceptionDetail1" class="rawExceptionDetails">
                        <pre class="rawExceptionStackTrace">Google.Apis.Auth.OAuth2.Responses.TokenResponseException: Error:&quot;invalid_grant&quot;, Description:&quot;Invalid JWT: Token must be a short-lived token (60 minutes) and in a reasonable timeframe. Check your iat and exp values and use a clock with skew to account for clock differences between systems.&quot;, Uri:&quot;&quot;&#xD;&#xA;   at Google.Apis.Auth.OAuth2.Requests.TokenRequestExtenstions.&lt;ExecuteAsync&gt;d__0.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\Requests\TokenRequestExtenstions.cs:line 52&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Auth.OAuth2.ServiceAccountCredential.&lt;RequestAccessTokenAsync&gt;d__23.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\ServiceAccountCredential.cs:line 212&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Auth.OAuth2.TokenRefreshManager.&lt;RefreshTokenAsync&gt;d__12.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\TokenRefreshManager.cs:line 129&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at Google.Apis.Auth.OAuth2.TokenRefreshManager.ResultWithUnwrappedExceptions[T](Task`1 task) in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\TokenRefreshManager.cs:line 174&#xD;&#xA;   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()&#xD;&#xA;   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)&#xD;&#xA;   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task&amp; currentTaskSlot)&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Auth.OAuth2.TokenRefreshManager.&lt;GetAccessTokenForRequestAsync&gt;d__10.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\TokenRefreshManager.cs:line 114&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Auth.OAuth2.ServiceAccountCredential.&lt;GetAccessTokenForRequestAsync&gt;d__24.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\ServiceAccountCredential.cs:line 235&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Auth.OAuth2.ServiceCredential.&lt;InterceptAsync&gt;d__26.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Auth\OAuth2\ServiceCredential.cs:line 163&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Http.ConfigurableMessageHandler.&lt;SendAsync&gt;d__63.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis.Core\Http\ConfigurableMessageHandler.cs:line 423&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()&#xD;&#xA;   at System.Net.Http.HttpClient.&lt;FinishSendAsyncBuffered&gt;d__58.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Requests.ClientServiceRequest`1.&lt;ExecuteUnparsedAsync&gt;d__30.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis\Requests\ClientServiceRequest.cs:line 231&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Requests.ClientServiceRequest`1.&lt;ExecuteAsync&gt;d__27.MoveNext() in C:\Apiary\2019-11-05.15-56-24\Src\Support\Google.Apis\Requests\ClientServiceRequest.cs:line 201&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Google.Apis.Requests.ClientServiceRequest`1.&lt;ExecuteAsync&gt;d__26.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()&#xD;&#xA;   at IntegrateWithCloud.Controllers.DefaultController.&lt;ReadAsync&gt;d__7.MoveNext() in D:\IntegrateWithCloud\IntegrateWithCloud\Controllers\DefaultController.cs:line 52&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at IntegrateWithCloud.Controllers.DefaultController.&lt;Get2&gt;d__5.MoveNext() in D:\IntegrateWithCloud\IntegrateWithCloud\Controllers\DefaultController.cs:line 32&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at IntegrateWithCloud.Controllers.DefaultController.&lt;GetAsync&gt;d__4.MoveNext() in D:\IntegrateWithCloud\IntegrateWithCloud\Controllers\DefaultController.cs:line 25&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at lambda_method(Closure , Object )&#xD;&#xA;   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.&lt;InvokeActionMethodAsync&gt;d__12.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.&lt;InvokeNextActionFilterAsync&gt;d__10.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State&amp; next, Scope&amp; scope, Object&amp; state, Boolean&amp; isCompleted)&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.&lt;InvokeInnerFilterAsync&gt;d__14.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.&lt;InvokeNextResourceFilter&gt;d__22.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State&amp; next, Scope&amp; scope, Object&amp; state, Boolean&amp; isCompleted)&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.&lt;InvokeFilterPipelineAsync&gt;d__17.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.&lt;InvokeAsync&gt;d__15.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Microsoft.AspNetCore.Builder.RouterMiddleware.&lt;Invoke&gt;d__4.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.&lt;Invoke&gt;d__7.MoveNext()</pre>
                    </div>
                </div>
            </li>
        </ul>
    </div>

    <div id="querypage" class="page">
        <p>No QueryString data.</p>
    </div>

    <div id="cookiespage" class="page">
        <p>No cookie data.</p>
    </div>
    <div id="headerspage" class="page">
        <table>
            <thead>
                <tr>
                    <th>Variable</th>
                    <th>Value</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>Accept</td>
                    <td>*/*</td>
                </tr>
                <tr>
                    <td>Accept-Encoding</td>
                    <td>gzip, deflate</td>
                </tr>
                <tr>
                    <td>Cache-Control</td>
                    <td>no-cache</td>
                </tr>
                <tr>
                    <td>Connection</td>
                    <td>keep-alive</td>
                </tr>
                <tr>
                    <td>Host</td>
                    <td>172.20.112.156</td>
                </tr>
                <tr>
                    <td>Postman-Token</td>
                    <td>1192bcf9-c5b7-4199-9ad4-b543ce51fae8</td>
                </tr>
                <tr>
                    <td>User-Agent</td>
                    <td>PostmanRuntime/7.19.0</td>
                </tr>
            </tbody>
        </table>
    </div>
    <script>
		//<!--
            (function (window, undefined) {
    "use strict";

    function ns(selector, element) {
        return new NodeCollection(selector, element);
    }

    function NodeCollection(selector, element) {
        this.items = [];
        element = element || window.document;

        var nodeList;

        if (typeof (selector) === "string") {
            nodeList = element.querySelectorAll(selector);
            for (var i = 0, l = nodeList.length; i < l; i++) {
                this.items.push(nodeList.item(i));
            }
        }
    }

    NodeCollection.prototype = {
        each: function (callback) {
            for (var i = 0, l = this.items.length; i < l; i++) {
                callback(this.items[i], i);
            }
            return this;
        },

        children: function (selector) {
            var children = [];

            this.each(function (el) {
                children = children.concat(ns(selector, el).items);
            });

            return ns(children);
        },

        hide: function () {
            this.each(function (el) {
                el.style.display = "none";
            });

            return this;
        },

        toggle: function () {
            this.each(function (el) {
                el.style.display = el.style.display === "none" ? "" : "none";
            });

            return this;
        },

        show: function () {
            this.each(function (el) {
                el.style.display = "";
            });

            return this;
        },

        addClass: function (className) {
            this.each(function (el) {
                var existingClassName = el.className,
                    classNames;
                if (!existingClassName) {
                    el.className = className;
                } else {
                    classNames = existingClassName.split(" ");
                    if (classNames.indexOf(className) < 0) {
                        el.className = existingClassName + " " + className;
                    }
                }
            });

            return this;
        },

        removeClass: function (className) {
            this.each(function (el) {
                var existingClassName = el.className,
                    classNames, index;
                if (existingClassName === className) {
                    el.className = "";
                } else if (existingClassName) {
                    classNames = existingClassName.split(" ");
                    index = classNames.indexOf(className);
                    if (index > 0) {
                        classNames.splice(index, 1);
                        el.className = classNames.join(" ");
                    }
                }
            });

            return this;
        },

        attr: function (name) {
            if (this.items.length === 0) {
                return null;
            }

            return this.items[0].getAttribute(name);
        },

        on: function (eventName, handler) {
            this.each(function (el, idx) {
                var callback = function (e) {
                    e = e || window.event;
                    if (!e.which && e.keyCode) {
                        e.which = e.keyCode; // Normalize IE8 key events
                    }
                    handler.apply(el, [e]);
                };

                if (el.addEventListener) { // DOM Events
                    el.addEventListener(eventName, callback, false);
                } else if (el.attachEvent) { // IE8 events
                    el.attachEvent("on" + eventName, callback);
                } else {
                    el["on" + type] = callback;
                }
            });

            return this;
        },

        click: function (handler) {
            return this.on("click", handler);
        },

        keypress: function (handler) {
            return this.on("keypress", handler);
        }
    };

    function frame(el) {
        ns(".source .collapsible", el).toggle();
    }

    function expandCollapseButton(el) {
        var frameId = el.getAttribute("data-frameId");
        frame(document.getElementById(frameId));
        if (el.innerText === "+") {
            el.innerText = "-";
        }
        else {
            el.innerText = "+";
        }
    }

    function tab(el) {
        var unselected = ns("#header .selected").removeClass("selected").attr("id");
        var selected = ns("#" + el.id).addClass("selected").attr("id");

        ns("#" + unselected + "page").hide();
        ns("#" + selected + "page").show();
    }

    ns(".rawExceptionDetails").hide();
    ns(".collapsible").hide();
    ns(".page").hide();
    ns("#stackpage").show();

    ns(".expandCollapseButton")
        .click(function () {
            expandCollapseButton(this);
        })
        .keypress(function (e) {
            if (e.which === 13) {
                expandCollapseButton(this);
            }
        });

    ns("#header li")
        .click(function () {
            tab(this);
        })
        .keypress(function (e) {
            if (e.which === 13) {
                tab(this);
            }
        });

    ns(".showRawException")
        .click(function () {
            var exceptionDetailId = this.getAttribute("data-exceptionDetailId");
            ns("#" + exceptionDetailId).toggle();
        });
})(window);
            //-->
    </script>
</body>

</html>